Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copied from https://github.com/BretFisher/node-docker-good-defaults/blob/master/Dockerfile
+
+FROM node:8.9.3
+
+# Create app directory
+RUN mkdir -p /edx/app
+
+ARG NODE_ENV=production
+ENV NODE_ENV $NODE_ENV
+
+ARG PORT=80
+ENV PORT $PORT
+EXPOSE $PORT 18400
+
+WORKDIR /edx
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
```

Comment:
I think I talked to you about this in person but just capturing it here, is there a reason we're copying this to the /edx dir rather than /edx/app?

Reply:
I am not sure, we should ask around.  Maybe @adamstankiewicz or @georgebabey might be able to provide insight.

Reply:
Tried changing this locally and all still seems to work. I will push an update.

Reply:
Makes sense to me to put into /edx/app

We have not been using docker for development but instead have been using our local machines so something we've never ran into